frontend/src/App.vue

```diff
@@ -28,6 +28,7 @@
 import LanguageSwitcher from '@/components/layout/LanguageSwitcher.vue'
 import VueI18n from '@/plugins/i18n'
 import { parseTemplate } from 'url-template'
+import { getEnv } from '@/dev-environment.js'
```

Comment:
if i did not use the function in some components,
the file (and specially the init of window.environment( was tree shaken and thus removed from the production build.

Maybe someone knows more about how to setup a javascript project properly